ibm_db2/README.md

```diff
@@ -170,23 +170,33 @@ See [service_checks.json][11] for a list of service checks provided by this inte
 
 ### Installing `ibm_db` client library offline
 
-If you're in an air gapped environment, or on a restricted network where it's not possible to run `pip install ibm_db==3.0.1`, you can install `ibm_db` using the following method:
+If you're in an air gapped environment, or on a restricted network where it's not possible to run `pip install ibm_db==<version>`, you can install `ibm_db` using the following method:
 
 **Note**: The following example assumes an Ubuntu machine, but the steps should also be similar on most operating systems.
 
-1. On a machine with network access, download [the source tarball][14].
+1. On a machine with network access, download the source tarballs for [the `ibm_db` library][14] and [the ODBC and CLI][16]. The ODBC and CLI are required to be downloaded separately because the `ibm_db` library requires them, but it cannot download them via `pip`.
 
    ```
    curl -Lo ibm_db.tar.gz https://github.com/ibmdb/python-ibmdb/archive/refs/tags/v3.1.0.tar.gz
```

Comment:
Should other places of ibm version be updated as well? Or would it be a static version for air gapped environments?

`curl -Lo ibm_db.tar.gz https://github.com/ibmdb/python-ibmdb/archive/refs/tags/v<version>.tar.gz`

Reply:
Thanks for the catch! It would indeed need to be different depending on Python 2 vs. Python 3.

Reply:
I'm going to keep it in for this example but just mention that the example is for 3.1.0 in the description, since I think people may be confused by the `v` if it is `v<version>`.